src/main/java/com/planning_system/services/TaskCommandService.java

```diff
@@ -1,146 +1,143 @@
 package com.planning_system.services;
 
-import com.planning_system.entity.Task;
-import com.planning_system.handlers.commands.Command;
-import com.planning_system.handlers.commands.OptionType;
-import com.planning_system.handlers.response.Response;
+import com.planning_system.controller.task.dto.TaskRequestDTO;
+import com.planning_system.controller.task.dto.TaskResponseDTO;
+import com.planning_system.entity.task.Task;
+import com.planning_system.entity.user.User;
+import com.planning_system.services.utility.TaskRequestDTOToTaskMapper;
+import com.planning_system.services.utility.TaskToTaskResponseDTOMapper;
 import com.planning_system.services.utility.TaskUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-import static com.planning_system.handlers.commands.OptionType.REJECTED_TASK;
+import static com.planning_system.controller.task.TaskParams.REJECTED_TASK;
 import static com.planning_system.services.messages.ServiceErrorMessages.ERROR;
-import static com.planning_system.services.messages.ServiceMessages.GET_ALL_REJECTED_TASKS;
-import static com.planning_system.services.messages.ServiceMessages.GET_ALL_TASKS;
-import static com.planning_system.services.messages.ServiceMessages.GET_TASK;
-import static com.planning_system.services.messages.ServiceMessages.NO_TASKS_YET;
-import static com.planning_system.services.messages.ServiceMessages.TASK_CREATED;
-import static com.planning_system.services.messages.ServiceMessages.TASK_DELETED;
-import static com.planning_system.services.messages.ServiceMessages.TASK_UPDATED;
-import static com.planning_system.services.utility.TaskUtil.parseId;
+import static com.planning_system.services.messages.ServiceErrorMessages.TASK_NOT_FOUND;
 
 /**
- * The class contains main business logic, responsible for {@link Task} processing and building {@link Response}.
+ * The class contains main business logic, responsible for {@link Task} processing
+ * and preparing data for response (filtering, sorting etc.)
  */
+@Service
 public class TaskCommandService {
 
     private final TaskService taskService;
-    private final RejectedTaskService rejectedTaskService;
+    private final TaskToTaskResponseDTOMapper taskToTaskResponseDTOMapper;
+    private final TaskRequestDTOToTaskMapper taskRequestDTOToTaskMapper;
 
+    @Autowired
```

Comment:
You can use lombok annotation, it will automatically inject all specified beans

https://projectlombok.org/features/constructor